[버그] 조회수 순으로 정렬하기 기능에서 에러 발생

```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'articleCommentController' defined in file 
[D:\Projects\GreenSumer_Back\out\production\classes\org\swyg\greensumer\controller\ArticleCommentController.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'articleCommentService' defined in file [D:\Projects\GreenSumer_Back\out\production\classes\org\swyg\greensumer\service\ArticleCommentService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'articleRepository' defined in org.swyg.greensumer.repository.ArticleRepository defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract org.springframework.data.domain.Page org.swyg.greensumer.repository.ArticleRepository.findByViews(org.springframework.data.domain.Pageable); Reason: Failed to create query for method public abstract org.springframework.data.domain.Page org.swyg.greensumer.repository.ArticleRepository.findByViews(org.springframework.data.domain.Pageable)! Method public abstract org.springframework.data.domain.Page org.swyg.greensumer.repository.ArticleRepository.findByViews(org.springframework.data.domain.Pageable) expects at least 1 arguments but only found 0. This leaves an operator of type SIMPLE_PROPERTY for property views unbound.; nested exception is java.lang.IllegalArgumentException: Failed to create query for method public abstract org.springframework.data.domain.Page org.swyg.greensumer.repository.ArticleRepository.findByViews(org.springframework.data.domain.Pageable)! Method public abstract org.springframework.data.domain.Page org.swyg.greensumer.repository.ArticleRepository.findByViews(org.springframework.data.domain.Pageable) expects at least 1 arguments but only found 0. This leaves an operator of type SIMPLE_PROPERTY for property views unbound.
```

다음과 같은 에러가 발생하여 조회수 순으로 정렬하는 기능은 보류하기로 함.

--- a/src/main/java/org/swyg/greensumer/domain/constant/SearchType.java
+++ b/src/main/java/org/swyg/greensumer/domain/constant/SearchType.java
@@ -7,8 +7,7 @@ public enum SearchType {
     CONTENT("본문"),
     ID("유저 ID"),
     NICKNAME("닉네임"),
-    HASHTAG("해시태그"),
-    VIEWS("조회수");
+    HASHTAG("해시태그");
 
     @Getter
     private final String description;

--- a/src/main/java/org/swyg/greensumer/repository/ArticleRepository.java
+++ b/src/main/java/org/swyg/greensumer/repository/ArticleRepository.java
@@ -23,7 +23,6 @@ public interface ArticleRepository extends
     Page<Article> findByUserAccount_UsernameContaining(String username, Pageable pageable);
     Page<Article> findByUserAccount_NicknameContaining(String nickname, Pageable pageable);
     Page<Article> findByHashtag(String hashtag, Pageable pageable);
-    Page<Article> findByViews(Pageable pageable);
 
     @Override
     default void customize(QuerydslBindings bindings, QArticle root){

--- a/src/main/java/org/swyg/greensumer/service/ArticleService.java
+++ b/src/main/java/org/swyg/greensumer/service/ArticleService.java
@@ -37,7 +37,6 @@ public class ArticleService {
             case ID -> articleRepository.findByUserAccount_UsernameContaining(searchKeyword, pageable).map(ArticleDto::from);
             case NICKNAME -> articleRepository.findByUserAccount_NicknameContaining(searchKeyword, pageable).map(ArticleDto::from);
             case HASHTAG -> articleRepository.findByHashtag("#" + searchKeyword, pageable).map(ArticleDto::from);
-            case VIEWS -> articleRepository.findByViews(pageable).map(ArticleDto::from);
         };
     }
 

